core/extraction/towns_fund_round_three.py

```diff
@@ -289,7 +289,7 @@ def extract_project(df_project: pd.DataFrame, project_lookup: dict, programme_id
     """
 
     # strip out everything except Section B
-    df_project = df_project.iloc[23:45, 4:]
+    df_project = df_project.iloc[23:45, 4:12]
```

Comment:
Out of interest what is this change for?

Reply:
One of the local authorities wrote comments outside the tables specified for input, the open ended range was reading their comments into the extract and creating ExtraColumn errors

Reply:
good stuff